modules/gateways/cardinity.php

```diff
@@ -443,16 +377,42 @@ function createWhmcsReturnArray($status, $rawData, $transactionId = '')
  */
 function addRemoteToken($invoiceId, $remoteTokenID)
 {
-    $user = Capsule::table('tblinvoiceitems')->select('userid')
+    if($remoteTokenID){
```

Comment:
in case this function gets called with empty tokenid, do not want to run the following queries.

Reply:
Based on the code, do we have any case when tokenId is not passed? My understanding is that recurring payments won't work in case remoteTokenId is not stored here. If that is the case, this added if statement will just skip storing silently - is that the intent?

Reply:
set it back to way it was